### PR TITLE
Configure qpid dispatch router on EL8

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Review by jturel — APPROVED** [review]
This looks pivotal in terms of katello-agent working on EL8 and I support the change.

**Comment by ehelms** [comment]
Forgot to fix the indent, got it now

**Comment by jturel** [comment]
Are there any tests that need to be updated that might not be making certain assertions based on an el7 check?

**Comment by ehelms** [comment]
I made one small update, but I think things are good as is for this change. I would like to add some additional testing in a follow up because I would like to pursue having dispatch router off by default and users turning on katello-agent support.

**Comment by jturel** [comment]
LGTM

File: spec/classes/foreman_proxy_content_spec.rb [hunk 1]
```diff
@@ -12,12 +12,6 @@
       end
 
       context 'with pulpcore' do
```
Comment (ekohl) [inline spec/classes/foreman_proxy_content_spec.rb:14]: This now becomes the same as `without parameters` so we can merge those two groups. https://github.com/theforeman/puppet-foreman_proxy_content/pull/325

**Pull request merged**